### ISSUE TITLE
After updating OSX to 10.10.3, Today-Scripts no longer appears installed.

### ISSUE THREAD

**Issue by gkopff** [SamRothCA/Today-Scripts#21]
If I launch the downloaded binary, it seems to install itself again and my configuration is correctly found and used.  (So it's a happy ending).

This may be nothing to do with Today-Scripts and everything to do with Apple, I don't know. :)

**Comment by zpeters** [comment]
I was able to get it back by downloading again, copying it to another folder and launching.  Unfortunately now i'm seeing it that it is back to it's default state and when i hover over the name i no longer see the "i" icon that lets you change the settings.

**Comment by zpeters** [comment]
Maybe i spoke too soon.  After a power off and back on i can see the widget along with my original info and the "i" icon.

**Comment by zpeters** [comment]
gkopff, what folder are you running the binary out of?

**Comment by gkopff** [comment]
I launched the binary from my Downloads folder.  (But I was under the impression that doing this had the effect of installing the widget, and that original binary is no longer required to be kept).

@zpeters if content disappears, you might like to look at this: https://github.com/SamRothCA/Today-Scripts/issues/10

**Comment by gkopff** [comment]
Off-topic, but @SamRothCA I love this thing :)  Best Widget Ever.

**Comment by rwilcox** [comment]
The widget disappeared for me too when upgrading to 10.10.3. A relaunch of the app (which I put in my Applications folder) brought it back.

**Comment by picree** [comment]
It's buggy.  The widgets stopped updating (I have multiple profiles in 10.10.4 if that matters), I can no longer edit the individual scripts, and the Info button for the overall app is gone.  So, I moved the app from Downloads to Applications.  Now the 3 scripts I had set up are gone. with no ability to add them back.  I haven't tried rebooting or re-adding to the Notification Center (which I assume would probably resolve the issues).

**Comment by picree** [comment]
I removed and re-added the app to the Notification Center.  Now nothing is clickable.  10.10.4.

**Comment by SamRothCA** [comment]
Yeah, El Captian users will experience the same symptoms at well. I have this fixed already as part of an ongoing overhaul, should have that out to you guys soon.

**Comment by rgkobashi** [comment]
I found two forks with different colors which works fine on 10.12.4: https://github.com/SamRothCA/Today-Scripts/issues/24#issuecomment-305772696
